components/Form/MultiStepForm.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Formik } from 'formik';
+import { getErrorMessage } from 'common/utils/api-utils';
+import { capitalizeFirstLetter } from 'common/utils/string-utils';
+import Button from 'components/Button/Button';
+import Form from 'components/Form/Form';
+import Alert from 'components/Alert/Alert';
+import styles from './MultiStepForm.css';
+
+class MultiStepForm extends React.Component {
+  static propTypes = {
+    // initialValues must be object where entire form's shape is described
+    initialValues: PropTypes.object.isRequired,
+
+    onFinalStepSuccess: PropTypes.func.isRequired,
+    steps: PropTypes.arrayOf(
+      PropTypes.shape({
+        render: PropTypes.func.isRequired,
+        validationSchema: PropTypes.object.isRequired, // specifically a Yup object shape
+        submitHandler: PropTypes.func,
+      }),
+    ).isRequired,
+  };
+
+  state = {
+    // eslint-disable-next-line react/destructuring-assignment
```

Comment:
Leftover from something that used to be here?

Reply:
💯

I ended up making a separate PR to merge _just_ the `MultiStepForm` component. In that PR, I removed [this line](https://github.com/OperationCode/front-end/pull/453/files).